config/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+# Contributor Covenant Code of Conduct
```

Comment:
The CoC is already in the root of the repo, looks like this was added by mistake?